.travis.yml

```diff
@@ -29,7 +33,14 @@ matrix:
       - python: "3.3"
         env: DJANGO=1.10
       - python: "3.3"
+        env: DJANGO=1.11
```

Comment:
python 3.3 is only tested against django 1.8, maybe it would be better to add it to the includes instead?